recipes/hyperspy/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set version = "1.0" %}
+
+package:
+  name: hyperspy
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/h/hyperspy/hyperspy-{{ version }}.tar.gz
+  md5: 4c50b20c2ff5f3d5f0fe609258e3d2b0
+
+    # url: https://github.com/hyperspy/hyperspy/archive/v{{ version }}.tar.gz
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - dill
+    - h5py
+    - ipython <5.0
+    - jupyter
+    - matplotlib >=1.2
+    - numpy
+    - qt
+    - scikit-image
+    - scikit-learn
+    - scipy
+    - setuptools
+    - statsmodels
+    - sympy
+    - natsort
+    - lxml
+    - ipyparallel
+    - dask
+    - traits >=4.5
+    - traitsui >=5.0
+    - requests
+    - ipywidgets >=5.0
+    - tqdm
+    - pip:
```

Comment:
With only one, and that one being a pure-python lib, it's OK here.  Generally discouraged because it makes debugging take longer.  Just a judgement call.  Could do as another PR.